_component_design/chips.md

```diff
@@ -17,27 +17,51 @@ helix-ui-javascript: false
 
 <section class="static-section" markdown="1">
 
-## Introduction
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+## When to use
+
+Use chips to indicate that a list of items is part of a different workflow. For example, use chips when users add or remove requestors to or from a ticket.
```

Comment:
This first sentence is confusing. I'm concerned that doesn't give enough info that a designer or dev would know that it's time to use a chip. 

I'm left with lots of weird questions.
e.g. 
when did a list come into the conversation?
Different workflow? different from what? 
What's a "requestor"?

Reply:
Agreed, I need assistance with this kind of content. How to proceed with making this better? Let's figure out a process...